symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/Message.java

```diff
@@ -0,0 +1,37 @@
+package com.symphony.bdk.core.service.message.model;
+
+import static java.util.Collections.emptyMap;
+
+import com.symphony.bdk.template.api.Template;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.apiguardian.api.API;
+
+@Getter
+@Setter
+@Accessors(fluent = true)
+@API(status = API.Status.EXPERIMENTAL)
+public class Message {
+
+  private String content;
+  private String data;
```

Comment:
`data` could also be `JsonNode` or any `Object` that we will then serialize to a JSON string value

Reply:
Done in the MessageBuilder class.